contentcuration/contentcuration/views/settings.py

```diff
@@ -44,13 +47,19 @@
 @browser_is_supported
 def settings(request):
     current_user = current_user_for_context(request.user)
+    channel_query = request.user.editable_channels \
+        .prefetch_related("editors") \
```

Comment:
If you're just annotating, no need to do the prefetch_related on editors, as the annotation will perform the join. Only need to do this if you are later going to reference the `editors` property off the model itself.